privacy/ccpa/policy_test.go

```diff
@@ -539,43 +625,55 @@ func TestBuildExtWrite(t *testing.T) {
 		{
 			description:   "Leaves Other Ext.Prebid Values",
 			noSaleBidders: []string{"a", "b"},
-			ext:           json.RawMessage(`{"prebid":{"other":"any"}}`),
-			expected:      json.RawMessage(`{"prebid":{"nosale":["a","b"],"other":"any"}}`),
+			ext:           json.RawMessage(`{"prebid":{"aliases":{"a":"b"},"supportdeals":true}}`),
+			expected:      json.RawMessage(`{"prebid":{"aliases":{"a":"b"},"supportdeals":true,"nosale":["a","b"]}}`),
 		},
```

Comment:
I guess the test now requires valid other values. Please use just `aliases` or `supportdeals`, not both. Same for the test below.